astroquery/noao/core.py

```diff
@@ -0,0 +1,74 @@
+"""
+Provide astroquery API access to OIR Lab Astro Data Archive (natica).
+
+This does DB access through web-services.
+"""
+
+import astropy.table
+from ..query import BaseQuery
+from ..utils import async_to_sync
+from ..utils.class_or_instance import class_or_instance
+from . import conf
+
+
+__all__ = ['Noao', 'NoaoClass']  # specifies what to import
+
+
+@async_to_sync
+class NoaoClass(BaseQuery):
+
+    TIMEOUT = conf.timeout
+    NAT_URL = conf.server
+    ADS_URL = f'{NAT_URL}/api/adv_search/fasearch'
+    SIA_URL = f'{NAT_URL}/api/sia/voimg'
+
+    def __init__(self, which='voimg'):
+        """ set some parameters """
+        self._api_version = None
+
+        if which == 'vohdu':
+            self.url = f'{self.NAT_URL}/api/sia/vohdu'
+        if which == 'voimg':
+            self.url = f'{self.NAT_URL}/api/sia/voimg'
+        else:
+            self.url = f'{self.NAT_URL}/api/sia/voimg'
```

Comment:
this whole block reads:
`self.url = f'{self.NAT_URL}/api/sia/voimg'`
no matter what.  I think you meant the middle `if` to be an `elif`?

The documentation here needs to be clearer; what does `which` mean?

Reply:
Good catch. Fixed locally for my next push.